apisix/plugins/echo.lua

```diff
@@ -52,6 +56,24 @@ local _M = {
 
 
 function _M.check_schema(conf)
+
+    if conf.headers then
```

Comment:
`headers` needs to be added in schema.

Reply:
yes will add them wanted to get an idea first weather the flow is correct.

Reply:
added